config/dependencies/envoy-gateway/gateway/gateway.yaml

```diff
@@ -0,0 +1,14 @@
+---
+apiVersion: gateway.networking.k8s.io/v1
+kind: Gateway
+metadata:
+  name: eg
```

Comment:
When we eventually merge the overarching feature we might want to see if we can deploy both `istio-ingressgateway` and envoy gateway `eg` with the same name/namespace so that all of the examples/quickstarts "just work" for both.

Reply:
yeah. we might need to pick a generic name for both. It cannot be either `istio-ingressgateway` or `eg`.